network/Networking.cpp

```diff
@@ -20,4 +20,26 @@ namespace Networking {
     { return GetOptionsDB().Get<int>("network.discovery-port"); }
     int MessagePort()
     { return GetOptionsDB().Get<int>("network.message-port"); }
+
+    AuthRoles::AuthRoles(const std::initializer_list<RoleType>& roles) {
+       for (RoleType r : roles) {
+           m_roles.set(r, true);
+       }
+    }
+
+    void AuthRoles::SetRole(RoleType role, bool value) {
+        m_roles.set(role, value);
```

Comment:
for these single-line functions, put the `{}` on the same line as the single line of code.

Reply:
Fixed.